charts/ceph-csi-cephfs/templates/csidriver-crd.yaml

```diff
@@ -9,3 +9,4 @@ metadata:
 spec:
   attachRequired: false
   podInfoOnMount: false
+  fsGroupPolicy: File
```

Comment:
@humblec can you provide details on why it should be File here, we have set fsGroupPolicy to ReadWriteOnceWithFSType in Rook for cephfs.

Reply:
@Madhu-1 dropping the PR for Rook to change that.. pretty much ready .. I will link the same here.

Reply:
@Madhu-1 https://github.com/rook/rook/pull/10503 address the same